src/plugins/discover/public/application/angular/context/api/anchor.test.ts

```diff
@@ -114,6 +116,28 @@ describe('context app', function () {
       });
     });
 
+    it('should update search source correctly when useFieldApi set to false', function () {
+      const searchSource = updateSearchSource(
+        savedSearchMock.searchSource,
+        'id',
+        [],
+        false,
+        indexPatternMock
+      );
+      expect(searchSource.getSearchRequestBody()).toMatchSnapshot();
```

Comment:
could we maybe set an explicit check for `track_total_hits` to be false, so that it's more immediately visible?

Reply:
sure! done!